### PR TITLE
TT-102 naming consistency

### PR DESCRIPTION
**Pull request by jesskn**
Changes 1-4 on: https://uktrade.atlassian.net/browse/TT-102

### CODE REVIEW

File: enrolment/templates/export-status-form.html [hunk 1]
```diff
@@ -10,10 +10,10 @@
         <div id="account-create-information">
             <p>To confirm that this is your company you must create a great.gov.uk account</p>
             <br>
-            <p><strong>An account will let you:</strong></p>
+            <p><strong>Creating an account means you can:</strong></p>
             <ul class="list">
-                <li class="list-item">Create a trade profile that will be promoted to international businesses</li>
```
Comment (SteveHhulmes) [inline enrolment/templates/export-status-form.html]: shouldn't it be lower case 'c'  ie create not Create

File: enrolment/templates/export-status-form.html [hunk 2]
```diff
@@ -10,10 +10,10 @@
...
-                <li class="list-item">Apply for export opportunities sourced by UK embassies worldwide</li>
```
Comment (SteveHhulmes) [inline enrolment/templates/export-status-form.html]: shouldn't it be apply for  not Apply for...

File: enrolment/templates/export-status-form.html (hunk 1 above)
Comment (jesskn) [reply enrolment/templates/export-status-form.html]: confirmed lower case

File: enrolment/templates/export-status-form.html (hunk 2 above)
Comment (jesskn) [reply enrolment/templates/export-status-form.html]: this had been removed

File: enrolment/templates/export-status-form.html (hunk 1 above)
Comment (jesskn) [reply enrolment/templates/export-status-form.html]: this also removed

**Pull request merged**